packages/inter-protocol/test/swingsetTests/psmUpgrade/test-psm-upgrade.js

```diff
@@ -0,0 +1,90 @@
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+
+import { assert } from '@agoric/assert';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { resolve as importMetaResolve } from 'import-meta-resolve';
+
+import { buildVatController } from '@agoric/swingset-vat';
+import { faV1BundleName } from './bootstrap-psm-upgrade.js';
+
+// so paths can be expresssed relative to this file and made absolute
+const bfile = name => new URL(name, import.meta.url).pathname;
+
+test('PSM service upgrade', async t => {
+  /** @type {SwingSetConfig} */
+  const config = {
+    includeDevDependencies: true, // test's bootstrap has some deps not needed in production
+    defaultManagerType: 'local',
+    bundleCachePath: 'bundles/',
+    bootstrap: 'bootstrap',
+    vats: {
+      bootstrap: {
+        // TODO refactor to use bootstrap-relay.js
```

Comment:
s/TODO/XXX/

Reply:
Would `bootstrap-relay.js` work here? I thought the custom bootstrap was required. I suspect `bootstrap-relay.js` would need some enhancement.

Reply:
what I had in mind is a test like the bootstrapTests that can use `EV` instead of `E` to make calls from an Ava context. but i'll just strike this comment. If I do get around to providing that, this test doesn't need to adopt that style and it's not tech debt to be in this style